WordPress/Classes/Models/Media.h

```diff
@@ -45,6 +45,8 @@ typedef NS_ENUM(NSUInteger, MediaType) {
 @property (nonatomic, strong, nullable) NSString *title;
 @property (nonatomic, strong, nullable) NSString *videopressGUID;
 @property (nonatomic, strong, nullable) NSNumber *width;
+@property (nonatomic, strong, nullable) NSError *error;
```

Comment:
Should we name this `uploadError` or do you think `error` will be descriptive enough by itself?

Reply:
The error can also be a processing/importing error, so I think error is good.